src/pages/projectSettings/tabs/NotificationsTab/SubscriptionField.tsx

```diff
@@ -0,0 +1,102 @@
+import React from "react";
+import styled from "@emotion/styled";
+import ExpandableCard from "@leafygreen-ui/expandable-card";
+import { Body } from "@leafygreen-ui/typography";
+
+import { Field } from "@rjsf/core";
+import { size } from "constants/tokens";
+import { triggers } from "constants/triggers";
+import { Subscriber } from "gql/generated/types";
+import { ResourceType, Trigger } from "types/triggers";
+import { SubscriptionSelect } from "./NotificationsRow";
+
+const getSentanceCase = (string: string) =>
+  string[0].toUpperCase() + string.substring(1).toLowerCase();
+
+export const getSubscriberType = (subscriber: Subscriber) => {
+  if (subscriber === undefined) {
+    return "";
+  }
+  let key: string = "";
+  Object.keys(subscriber).forEach((k) => {
+    if (subscriber[k] !== null) {
+      key = k;
+      return k;
+    }
+    return null;
+  });
+  return key;
+};
```

Comment:
I think returning stuff before the function ends negates the purpose of having the variable `key`. Maybe this would work?
```suggestion
  let key: string = "";
  Object.keys(subscriber).forEach((k) => {
    if (subscriber[k] !== null) {
      key = k;
    }
  });
  return key;
};
```